core/src/time/drift_correction/ticks.rs

```diff
@@ -87,14 +87,14 @@ mod test {
   pub fn ticks_drift_correction_next() {
     let mut correction = TicksDriftCorrection::new(Signature::new(4, 4), Tempo::new(60), 44100);
     let ticks = correction.next(100);
-    assert_eq!(ticks, TicksTime::new(628));
+    assert_eq!(ticks, TicksTime::new(4608000));
     let ticks = correction.next(100);
-    assert_eq!(ticks, TicksTime::new(628));
+    assert_eq!(ticks, TicksTime::new(4608000));
     let ticks = correction.next(100);
-    assert_eq!(ticks, TicksTime::new(627));
+    assert_eq!(ticks, TicksTime::new(4608000));
     let ticks = correction.next(100);
-    assert_eq!(ticks, TicksTime::new(628));
+    assert_eq!(ticks, TicksTime::new(4608000));
     let ticks = correction.next(100);
-    assert_eq!(ticks, TicksTime::new(627));
+    assert_eq!(ticks, TicksTime::new(4608000));
```

Comment:
Hmmm, I see we'll need to pick some other example of Signature/Tempo/SampleRate to force some error correction (you might want to use the ones from the previous test). Given the increased resolution the correction doesn't trigger in the test anymore. The trick of this test is to show that an error gets accumulated and corrected by seeing a different value every n steps (i.e. 628, 628, 627, 628, ...).

Reply:
Look, this is the test adapted to the new resolution. Felt lazy to create a PR on your PR 😉 Could you please modify it in your branch ? Then everything will be ready for merging.
```rust
  #[test]
  pub fn ticks_drift_correction_next() {
    let mut correction = TicksDriftCorrection::new(Signature::new(6, 11), Tempo::new(131), 44100);
    for _ in 0..4 {
      let ticks = correction.next(1000);
      assert_eq!(ticks, TicksTime::new(36584727));  
    }
    let ticks = correction.next(1000);
    assert_eq!(ticks, TicksTime::new(36584728));
  }
```

Reply:
Thanks Chris, I pushed that change. Github is saying there are some conflicts in
- core/src/time/drift_correction/ticks.rs
- core/src/time/ticks.rs

The information it shows is a little confusing for me so I'm not sure how to fix that bit.. It looks like the conflict is between your audio-lockfree branch and my issue branch. Not sure why but maybe I did the PR wrong and didn't request to merge with your master branch? 🤷‍♂

Reply:
@Engid fixed. I changed to master branch.